corehq/apps/reports/analytics/esaccessors.py

```diff
@@ -382,10 +380,8 @@ def _chunked_get_form_counts_by_user_xmlns(domain, startdate, enddate, user_ids=
                      TermsAggregation('app_id', 'app_id').aggregation(
                          TermsAggregation('xmlns', 'xmlns.exact')
                      )
-                 )
-             )
-             .size(0)
-    )
+                 ))
```

Comment:
I think the _before_ is better as it lines up with the function.

Reply:
I agree, and I can play around with the formatting to see if I can get something more readable that still adheres to the rules, but I think the benefits of having all our code follow the same ruleset outweighs the individual instances that would be better served with custom formatting -- although I could explicitly ignore the linter here if you think there's really that big of a difference?

Reply:
Nah, I'm just anal-retentive seeing yellow and red on my screen in visual studio code. Reports are my baby now, and I just wanted to set a concrete foundation to build off of.

Reply:
If you are going to do things like this on a large scale, I would advocate for using yapf since that will give the most consistent results, and we have a standard configuration: 

https://github.com/dimagi/commcare-hq/blob/74bc31910f692126f03c46a350ab8ae5700f87dd/.style.yapf

Reply:
Unfortunately, (assuming I have my configuration working correctly), yapf and flake8 don't like each other's settings. Formatting this file with yapf causes flake8 to bark about closing brackets not matching visual indentations. I'm going to stick to the initial version that fixed the flake8 warnings, as I just want to set a baseline here.